src/Medidata.ZipkinTracer.Core/SpanCollector.cs

```diff
@@ -41,12 +33,12 @@ public virtual void Collect(Span span)
 
         public virtual void Start()
         {
-            spanProcessor.Start();
+            SyncHelper.ExecuteSafely(syncObj, () => !IsStarted, () => { spanProcessor.Start(); IsStarted = true; });
```

Comment:
@klettier Can we bring ExecuteSafely() method at the SpanProcessorTaskFactory level? I think it has the similar logic for IsStarted there. it checks the spanProcessorTaskInstance if it is already instantiated.